pymongo/asynchronous/encryption.py

```diff
@@ -905,6 +935,7 @@ async def _encrypt_helper(
                 contention_factor=contention_factor,
                 range_opts=range_opts_bytes,
                 is_expression=is_expression,
+                text_opts=text_opts_bytes,
```

Comment:
Passing `text_opts` even when it's None will fail on old versions of pymongocrypt. So we need to bump the min version or avoid passing text_opts by default.